### ISSUE TITLE
Future special effects: Selection according to selected Lightworks version

### ISSUE THREAD

**Issue by FxSchrauber** [fx-planet/fxplanet#4]
Lightworks 14.5 added new automatic effect parameters.
Also, image files can be installed together with special new effect.

Not all future effects will be compatible with older versions of Lightworks.
I think if the user could select the used Lightworks version, and maybe also the operating system, it would be helpful if only the relevant effects / download links would be displayed to him.

**Comment by marcinn** [comment]
Yes, that's good idea. I'm thinking about `@platform` attr for cross-platform compatibility , but `@compat` (for specific "host" version) can be also helpful.

**Comment by jwrl1** [comment]
For what it's worth, my preference is to make any effect I write backwards compatible.  It usually isn't too hard.  For example, today I have been working with a flicker generator that uses a time parameter.  By declaring _Length=1/0, it will still return the correct value in 14.5, but will return 1.0 in 14.0 and earlier.  Because I calculate time = _Progress * _Length, time will equal _Progress in older versions but clip time in 14.5 and up.  By allowing the user to scale time the difference in behaviour can be compensated for.
